.travis.yml

```diff
@@ -19,23 +24,41 @@ before_script:
   - make
   - sudo make install
   - cd ../..
-  #build python bindings
+
+  # build python bindings python 2
   - cd python_orocos_kdl
-  - mkdir build
-  - cd build
-  - cmake -DCMAKE_CXX_FLAGS:STRING="-Wall" -DCMAKE_BUILD_TYPE=${OROCOS_KDL_BUILD_TYPE} ./..
-  #compile and install python bindings
+  - mkdir build2
+  - cd build2
+  - export ROS_PYTHON_VERSION=2
+  - cmake -DCMAKE_CXX_FLAGS:STRING="-Wall" -DCMAKE_BUILD_TYPE=${OROCOS_KDL_BUILD_TYPE} -DBUILD_PYKDL_PYBIND11=${BUILD_PYKDL_PYBIND11} ./..
+  # compile and install python bindings python 2
+  - make
+  - sudo make install
+  - export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
+  - sudo ldconfig
+  - cd ../..
+
+  # build python bindings python 3
+  - cd python_orocos_kdl
+  - mkdir build3
+  - cd build3
+  - export ROS_PYTHON_VERSION=3
+  - cmake -DCMAKE_CXX_FLAGS:STRING="-Wall" -DCMAKE_BUILD_TYPE=${OROCOS_KDL_BUILD_TYPE} -DBUILD_PYKDL_PYBIND11=${BUILD_PYKDL_PYBIND11} ./..
+  # compile and install python bindings python 3
```

Comment:
I haven't looked into detail yet, but I guess the above steps are only required to make travis happy so it can execute the tests?

Reply:
Yes, I could create separate jobs for python 2 and 3, but that would double the number of jobs. Therefore I chose to combine python 2 and 3 in one job, as they also build against the same KDL version.